saphana-plugin/widgets/SapHana-action.json

```diff
@@ -0,0 +1,87 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "display-name": "SAP HANA Execute",
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Driver Name",
+          "name": "jdbcPluginName",
+          "widget-attributes": {
+            "default": "sap"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Host",
+          "name": "host",
+          "widget-attributes": {
+            "default": "localhost"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Port",
+          "name": "port",
+          "widget-attributes": {
+            "default": "39017"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Database",
+          "name": "database"
+        },
+        {
+          "widget-type": "textarea",
+          "label": "Database Command",
```

Comment:
nit: If this query uses SQL based language you could use `sql-editor` as widget. The only advantage is it has syntax highlighting.

Reply:
done